lib/fizzy/bytes.hpp

```diff
@@ -1,3 +1,7 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2019-2020 The Fizzy Authors.
+// Licensed under the Apache License, Version 2.0.
```

Comment:
@chfast suggested https://spdx.org/ids-how

So would we do
```
// SPDX-License-Identifier: Apache-2.0
```
or

```
// Licensed under the Apache License, Version 2.0.
// SPDX-License-Identifier: Apache-2.0
```

Reply:
The first one.